doc/conf.py

```diff
@@ -95,6 +97,18 @@ def _check_deps():
   'pandas': ('http://pandas.pydata.org/pandas-docs/stable', None)
   }
 
+explicit_order_folders = ['../examples/pyplots',
+                          '../examples/statistics',
+                          '../examples/color',
+                          '../examples/pylab_examples',
```

Comment:
whoah way to go @NelleV ! I didn't know we were so close to moving everything outta that folder. I'll rebase and see if it's gone now.